core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -2395,6 +2395,8 @@ object TestUtils extends Logging {
     )
 
     def unexpectedThreads: Set[String] = {
+      val x = Thread.getAllStackTraces.asScala.filter(entry => unexpectedThreadNames.exists(s => entry._1.getName.contains(s)))
```

Comment:
I guess it's a leftover from some debugging?

Reply:
yup :D good catch I was debugging some hanging thread issues